src/components/Main/Main.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
```

Comment:
Main.js is considered a view not a presentational component. Go ahead and makes a views folder in your src and transfer the entire Main folder into that.